### ISSUE TITLE
sscan without block

### ISSUE THREAD

**Issue by mperham** [redis-rb/redis-client#24]
I have this code in Sidekiq:

```ruby
       procs = conn.sscan_each("processes").to_a
 ```

This worked in 0.4.0 and broke in the latest version. Recall that I need code which works with both redis-rb and redis-client.

```
ArgumentError: can't issue an empty redis command
    /Users/mperham/.gem/ruby/3.0.3/gems/redis-client-0.5.0/lib/redis_client/command_builder.rb:44:in `generate!'
    /Users/mperham/.gem/ruby/3.0.3/gems/redis-client-0.5.0/lib/redis_client.rb:245:in `sscan'
    /Users/mperham/src/sidekiq/lib/sidekiq/api.rb:972:in `each'
    /Users/mperham/src/sidekiq/lib/sidekiq/api.rb:972:in `to_a'
    /Users/mperham/src/sidekiq/lib/sidekiq/api.rb:972:in `block in each'
    /Users/mperham/src/sidekiq/lib/sidekiq.rb:156:in `block in redis'
```

**Comment by casperisfine** [comment]
Thank you for the bug report. We were lacking coverage on `scan` methods with no extra arguments other than the key.

I released a 0.5.1 with the fix.

**Comment by mperham** [comment]
Verified. Thank you!
